lib/schema-compilers.js

```diff
@@ -43,13 +43,13 @@ function ValidatorCompiler (externalSchemas, options, cache) {
 
   Object.values(externalSchemas).forEach(s => ajv.addSchema(s))
 
-  return (method, url, httpPart, schema) => {
+  return ({ schema, method, url, httpPart }) => {
     return ajv.compile(schema)
   }
 }
 
 function SerializerCompiler (externalSchemas) {
-  return function (method, url, httpStatus, schema) {
+  return function ({ schema, method, url, httpStatus }) {
```

Comment:
I would prefer we kept these as positional arguments. I have a hunch that this destructuring is more costly than positional arguments, and this is somewhat a fast path.

Reply:
Is there a way to verify the hunch? Would one of these be better?

```js
function foo (args) {
  const { schema, method, url, httpStatus = args
}

function bar (args) {
  const compiledSchema = compiler(args.schema)
}
```

Reply:
My bad, this is not a hot path at all.